specs/shell/make_a_dir_and_change_into.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: Make a directory and change into it
+command: mkdir {{dirname}} && cd $_
+tags:
+  - mkdir
+  - cd
```

Comment:
Could we use the `shell` tag instead? I'm worried about adding `mkdir` and `cd` tags since this would be only workflow for both of those tags.

Reply:
sure, makes sense